copilot/fastapi-proxy/src/service/router.ts

```
@@ -0,0 +1,44 @@
+router.use(express.json());
+
+router.all('*', async (req, res) => {
+  const targetUrl = `http://fastapi-service:8000${req.path}`;
+  
+  try {
+    // Forward original headers but ensure content-type is set for JSON body
+    const headers = { ...req.headers };
+    if (req.body) {
+      headers['Content-Type'] = 'application/json';
+    }
+    delete headers.host; // Remove host header as it would be incorrect for the target
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)